contact_manager.py

```diff
@@ -163,11 +168,9 @@ def write_db_to_file(self, db, out_path):
             #for each contact
             for contact_name in db:
                 #write the values of each in the correct format
-                outf.write(self.read_contact(db, contact_name.upper()))
-            #DEBUG/NOT WORKING: clear any blank lines
-            for line in outf:
-                if not line.isspace():
-                    outf.write(line)
+                this_contact = self.extract(db, contact_name.upper())
+                outf.write(self.format_contact_info(this_contact))
+                outf.write('\n')
```

Comment:
If this newline is removed there is an error when we use `write_db_to_file` and then try to open again.